src/main/java/org/skriptlang/skript/lang/script/ScriptWarning.java

```diff
@@ -75,16 +74,15 @@ public boolean isDeprecated() {
 	}
 
 	/**
-	 * Returns the deprecation message of this warning, or null if the warning isn't deprecated. 
-	 * @return The deprecation message.
+	 * @return The deprecation message of this warning, or null if the warning isn't deprecated.
 	 * @see #isDeprecated()
 	 */
-	public @UnknownNullability String getDeprecationMessage() {
+	public String getDeprecationMessage() {
 		return deprecationMessage;
 	}
 
 	/**
-	 * Prints the given message using {@link Skript#warning(String)} iff the current script does not suppress deprecation warnings.
+	 * Prints the given message using {@link Skript#warning(String)} if the current script does not suppress deprecation warnings.
```

Comment:
iff is intentional, it means if and only if.

Reply:
It's a user readable description.

Reply:
Then write `if and only if`